src/utils/index.js

```diff
@@ -131,14 +131,25 @@ var isIOS = module.exports.isIOS = function () {
 };
 
 /**
- * Checks mobile device orientation
- * @return {Boolean} True if landscape orientation
+ * Checks mobile device orientation.
+ * @return {Boolean} True if landscape orientation.
  */
 module.exports.isLandscape = function () {
   return window.orientation === 90 || window.orientation === -90;
 };
 
 /**
+ * Returns whether we can safely listen to this keyboard event.
+ * @returns {Boolean} True if we can.
```

Comment:
it's capitalised because a sentence - and we do elsewhere. I would say `true`. will change.